crates/cairo-annotations/src/trace_data.rs

```diff
@@ -146,6 +147,46 @@ pub enum DeprecatedSyscallSelector {
     Sha256ProcessBlock,
 }
 
+impl DeprecatedSyscallSelector {
+    #[must_use]
+    pub fn all() -> &'static [Self] {
```

Comment:
We could derive `[VariantArray`](https://docs.rs/strum_macros/latest/strum_macros/derive.VariantArray.html) 

And do:

```rust
impl DeprecatedSyscallSelector {
    #[must_use]
    pub fn all() -> &'static [Self] {
        Self::VARIANTS
    }
}
```

Or just directly use `VARIANTS` constant